From 5bc785dd5ee3473edad1e504717343fd217a1a63 Mon Sep 17 00:00:00 2001
From: Davy <davy.triponney@gmail.com>
Date: Thu, 2 Jan 2025 16:27:15 +0100
Subject: [PATCH] Build flag

Fix #245
---
 sources/polyphone.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sources/polyphone.pro b/sources/polyphone.pro
index 065c15a8..7516f5da 100644
--- a/sources/polyphone.pro
+++ b/sources/polyphone.pro
@@ -48,7 +48,7 @@ TRANSLATIONS = polyphone_en.ts \
 PRECOMPILED_HEADER = precompiled_header.h
 CONFIG += lrelease embed_translations precompiled_header
 QMAKE_LRELEASE_FLAGS = -nounfinished -removeidentical
-QMAKE_CXXFLAGS += -std=c++17 -O3 -msse2
+QMAKE_CXXFLAGS += -std=c++17 -O3
 #QMAKE_CXXFLAGS += -std=c++17 -O3 -mcpu=native -march=native
 
 QT += core gui printsupport svg network #testlib
